SmartDeviceLink/SDLFocusableItemLocator.m

```diff
@@ -6,6 +6,7 @@
 //
 
 #import <Foundation/Foundation.h>
+#import <simd/simd.h>
```

Comment:
The `rect` property of the `SDLHapticRect` still needs to be scaled in the `sdl_sendHapticRPC` method in this class as the Android PR does. Also, test cases still need to be added to the `SDLHapticManagerSpec` to make sure the `rect` of the resulting `SDLHapticRect` is scaled correctly.

Reply:
fixed, rect tests added.